applications/polyp_detection/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project(polyp_detection NONE)
+
+# download model
+add_custom_target(download_polyp_detection_rt_detr_model ALL
+    COMMAND ngc registry model download-version "nvstaging/holoscan/polyp_detection_rt_detr_model:20250304" --dest "${HOLOHUB_DATA_DIR}/polyp_detection"
```

Comment:
Model is now on NGC: https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/models/polyp_detection_rt_detr_model